content/docs/2.5/concepts/scaling-jobs.md

```diff
@@ -95,6 +95,18 @@ The actual number of jobs could exceed the limit in a short time. However, it is
 
 This optional property specifies the name of container in the Job, from which KEDA should try to get environment properties holding secrets etc. If it is not defined it, KEDA will try to get environment properties from the first Container, ie. from `.spec.JobTargetRef.template.spec.containers[0]`.
 
+---
+
+```yaml
+  rolloutStrategy: default # Optional. Default: default. Which Rollout Strategy KEDA will use. 
+```
+
+This optional property specifies the rollout strategy KEDA will use while updating an existing scaledJob.
+Possible values are `default` or `gradual`. \
+When using the `default` rolloutStrategy, KEDA will terminate existing Jobs whenever a scaledJob is being updated. Then, it will recreate those Jobs with the latest specs. \
+On the `gradual` rolloutStartegy, whenever a scaledJob is being updated, KEDA will not delete existing Jobs. Only new Jobs will be created with the latest specs. 
```

Comment:
```suggestion
This optional property specifies the rollout strategy KEDA will use while updating an existing ScaledJob.
Possible values are `default` or `gradual`. \
When using the `default` rolloutStrategy, KEDA will terminate existing Jobs whenever a ScaledJob is being updated. Then, it will recreate those Jobs with the latest specs. \
On the `gradual` rolloutStartegy, whenever a ScaledJob is being updated, KEDA will not delete existing Jobs. Only new Jobs will be created with the latest specs. 
```

Reply:
Could you please move this section below `maxReplicaCount` section (before `scalingStrategy`). Thanks!